src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx

```diff
@@ -76,9 +80,46 @@ class OrganizationIntegrations extends AsyncComponent<
     organization: SentryTypes.Organization,
   };
 
+  componentDidMount() {
+    // eslint-disable-next-line react/no-did-mount-set-state
+    this.setState({
```

Comment:
@NisanthanNanthakumar  We shouldn't call `setState` in `componentDidMount` just to set static default values. We have `getDefaultState` to handle that.